tests/test_utils.py

```diff
@@ -190,15 +190,16 @@ def test_can_undo_fp16_conversion(self):
         model = extract_model_from_parallel(model, keep_fp32_wrapper=False)
         _ = pickle.dumps(model)
 
-    @require_cuda
+    @require_triton
```

Comment:
I think this should be fine but I would love to have your opinion @BenjaminBossan. triton should be installed when we install torch.